### PR TITLE
Add fileId to organization

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

**Comment by dara123** [comment]
devs please review

**Comment by XaosSintez** [comment]
May be `logoFileId` is better? Because `fileId` is not very clear field name

File: spec/definitions/Organization.yaml [hunk 1]
```diff
@@ -43,6 +43,10 @@ properties:
     description: The organization postal code
     type: string
     maxLength: 10
+  fileId:
+    description: The organization logo (this logo will appear on your invoice)
+    type: string
+    maxLength: 50
```
Comment (colinbird) [inline spec/definitions/Organization.yaml]: Should it be like
```
  fileId:
    description: Linked File object id for organization logo (to be shown on invoices)
    allOf:
      - $ref: "#/definitions/ResourceId"```

**Comment by dara123** [comment]
My bad we want to add

**Pull request closed without merging**